src/core/Tree/Tree.tsx

```diff
@@ -0,0 +1,216 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { CommonProps, useTheme } from '../utils';
+import '@itwin/itwinui-css/css/tree.css';
+import cx from 'classnames';
+
+export const TreeContext = React.createContext<
```

Comment:
In `ColorPicker` you had it in separate file.. I see MenuItem has it inside file, but its not exported anywhere. So need to align.

Reply:
Not sure why `ColorPicker` has it in a separate file but I think that was a special circumstance. But `TreeContext` does need to be exported so that it can be used in `TreeNode`